### PR TITLE
DOC: Update roadmap with more items

### PR DESCRIPTION
**Pull request by larsoner**
Adds some stuff to our roadmap

### CODE REVIEW

File: examples/datasets/opm_data.py [hunk 1]
```diff
@@ -93,7 +93,7 @@
 
 # Fit dipoles on a subset of time points
 with mne.use_coil_def(coil_def_fname):
-    dip_opm, _ = mne.fit_dipole(evoked.copy().crop(0.015, 0.080),
+    dip_opm, _ = mne.fit_dipole(evoked.copy().crop(0.040, 0.080),
```
Comment (agramfort) [inline examples/datasets/opm_data.py:96]: why did you change this?

**Review by agramfort — APPROVED** [review]
besides LGTM

File: examples/datasets/opm_data.py (hunk 1 above)
Comment (larsoner) [reply examples/datasets/opm_data.py:96]: To make it run a little bit faster. The earlier outputs aren't really used anyway

**Comment by agramfort** [comment]
Thx @larsoner
